.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+name: 'GH Actions CI'
+
+on:
+  push:
+    branches: ['*master', '*0.[0-9].x']
```

Comment:
This might need to be  `[0-9]?` for when we get to multiple digits

Reply:
Oh. True.

Reply:
Done